functions/streams.js

```diff
@@ -27,12 +28,27 @@ exports.handler = async (event, context, callback) => {
     }
   )
 
+  const streamerFilter = (streamer) => {
+    let allowStreamer = true;
+    if (GAME_TITLE) {
+      console.log(GAME_TITLE)
+      allowStreamer = streamer.game_name === GAME_TITLE && allowStreamer;
+    }
+    if (STREAM_TITLE_REGEX) {
+      const regex = new RegExp(STREAM_TITLE_REGEX);
+      allowStreamer = streamer.title.match(regex) && allowStreamer;
+    }
```

Comment:
```suggestion
    if (STREAM_TITLE_FILTER) {
      allowStreamer = streamer.title.toLowerCase().includes(STREAM_TITLE_FILTER.toLowerCase()) && allowStreamer;
    }
```

Probably makes sense to do the comparison in lowercase